contracts/src/access/ownable.rs

```diff
@@ -124,6 +114,23 @@ impl Ownable {
 }
 
 impl Ownable {
+    /// Checks if the [`msg::sender`] is set as the owner.
```

Comment:
Added! Unfortunately I don't think we can make `Ownable2Step` inherit from `IOwnable` since we cannot expose two public `impl` right? That would have saved a bunch of doc repetition

Reply:
Yes, unfortunately we cannot do this because of a single `#[public]` per contract. But still you can write sth like reexport of [`IOwnable::*`]